packages/document/docs/en/plugin/system/introduction.mdx

```diff
@@ -4,7 +4,7 @@ The plugin system is a crucial part of Rspress, which allows you to easily exten
 
 The overall architecture of Rspress is shown in the figure below:
 
-![Rspress Architecture](https://lf3-static.bytednsdoc.com/obj/eden-cn/uhbfnupenuhf/edenx-doc/arch-en.png)
+![Rspress Architecture](https://assets.rspack.dev/rsdoctor/css-extract-loader.png)
```

Comment:
Is this intented?

Reply:
Yep, I imported the image file as a asset module right on the [zh translation](https://github.com/web-infra-dev/rspress/blob/feat/rsbuild-image/packages/document/docs/zh/plugin/system/introduction.mdx). So that we can compare the different of their behaviours.
I updated the PR summary. You can see more context there.

Reply:
The image is intented to be `Rspress Architecture`, but the new one `css-extract-loader` is definitely not.